src/transformers/models/luke/tokenization_luke.py

```diff
@@ -85,7 +85,9 @@
 
                 `What are attention masks? <../glossary.html#attention-mask>`__
             return_overflowing_tokens (:obj:`bool`, `optional`, defaults to :obj:`False`):
-                Whether or not to return overflowing token sequences.
+                Whether or not to return overflowing token sequences.If a pair of sequences of input ids ( or a batch
+                of pair ) is provided with :obj:`truncation_strategy = longest_first` or `True`, It raises an error
+                instead of returning overflowing tokens.
```

Comment:
```suggestion
                Whether or not to return overflowing token sequences. If a pair of sequences of input ids (or a batch
                of pairs) is provided with :obj:`truncation_strategy = longest_first` or :obj:`True`, an error is raised
                instead of returning overflowing tokens.
```